include/wx/fontutil.h

```diff
@@ -179,6 +179,9 @@ class WXDLLIMPEXP_CORE wxNativeFontInfo
     
     void RealizeResource() const;
 private:
+    // the PostScript names reported in macOS start with a dot for System Fonts, this has to be corrected
+    // otherwise round-trips are not possible, resulting in a Times Fallback
+    void AdjustPostScriptName();
```

Comment:
This class doesn't really follow this convention (probably because it predates it), but normally this should be
```suggestion
    void OSXAdjustPostScriptName();
```

Reply:
I'm fine with that change